### PR TITLE
add into string methods for single-float and double-floats

### PR DESCRIPTION
**Pull request by yaktender**

### CODE REVIEW

File: library/arith.lisp [hunk 1]
```diff
@@ -692,6 +692,17 @@ The fields are defined as follows:
       (lisp String (z)
         (cl:format cl:nil "~D" z))))
 
+  (define-instance (Into Single-Float String)
+    (define (into z)
+      (lisp String (z)
+        (cl:format cl:nil "~D" z))))
```
Comment (stylewarning) [inline library/arith.lisp:698]: `~D` is for (Decimal) printing of integers.

You might instead opt for `prin1-to-string`, which will try to write a `read`able string

**Review by stylewarning — CHANGES_REQUESTED** [review]

File: library/arith.lisp (hunk 1 above)
Comment (yaktender) [reply library/arith.lisp]: ok updated

**Review by aijony — COMMENTED** [review]
Make sure you are on the latest branch as `library/arith.lisp` has been moved. Also, into instances dealing with strings have been moved to `library/string.lisp`.

**Comment by eliaslfox** [comment]
I had to open a new pr to rewrite the commit history. See https://github.com/coalton-lang/coalton/pull/633.

**Pull request closed without merging**